src/01-general/users.apib

```diff
@@ -40,6 +40,8 @@ Get the current authenticated user.
             + time_zone: `Europe/Brussels` (string)
             + preferences (object)
                 + invoiceable: true (boolean)
+                + time_tracking_freezing: true (boolean)
+                + time_tracking_freezing_cutoff_days: 7 (number)
```

Comment:
Q: what if we (in the future) want to freeze time trackings after i.e. 8 hours? How can we reflect this in the `users.me` response?

Reply:
I also would go for a value/unit object here.